workflow-templates/git-slice-v2.yml

```
@@ -44,12 +44,32 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'workflow_dispatch' }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out repository
+        uses: actions/checkout@v2
         with:
           fetch-depth: 1
+
+      - name: Setup Git
+        run: |
+          git config --global user.email ${{ secrets.EMAIL }}
+          git config --global user.name  ${{ secrets.USERNAME }}
+          git config --global pull.rebase true
+          
+      - name: Merge main/master
+        run: |
+          git fetch -f
+          git branch
+          git checkout ${{ github.event.inputs.branch }}
+          git pull origin ${{ github.event.inputs.branch }}
+          git merge origin/master
```

```suggestion
          git pull origin ${{ github.event.inputs.branch }}
          git merge origin/${{ env.GIT_SLICE_DEFAULT_BRANCH }}
```
I'm considering getting it some other way, so that we can remove this variable entirely